webrick/upstart/tasks/RedHat.yml

```diff
@@ -0,0 +1,10 @@
+---
+- name: systemd unit - RedHat
+  template:
+    dest: "{{ webrick_systemd_unit }}"
+    src: systemd.service.j2
```

Comment:
upstart and systemd are two different service systems. I would suggest to create a different role for systemd or rename this role.

Next Ubuntu will have systemd as well. So we will have to support it very well.
Maybe we also find a way to recreate the userjob system. Which would allow a restart of the service without root rights.